direct/data/transforms.py

```diff
@@ -344,7 +406,16 @@ def ifftshift(data: torch.Tensor, dim: Tuple[Union[str, int], ...] = None) -> to
 
 
 def complex_multiplication(input_tensor: torch.Tensor, other_tensor: torch.Tensor) -> torch.Tensor:
-    """Multiplies two complex-valued tensors. Assumes input tensors are complex (last axis has dimension 2).
```

Comment:
Do not remove comment

Reply:
Moved it to `origin_complex_multiplication`

Reply:
Keep comment unchanged this way please